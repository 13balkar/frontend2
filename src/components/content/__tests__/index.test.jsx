import * as React from 'react';
import { render } from '@testing-library/react';
import Content from '../';
jest.mock('react-router-dom', () => ({
  useNavigate: () => jest.fn()
}));

describe('Content', () => {
  it('should render the content', () => {
    const { asFragment } = render(<Content />);
    expect(asFragment()).toMatchSnapshot();
  });
});
