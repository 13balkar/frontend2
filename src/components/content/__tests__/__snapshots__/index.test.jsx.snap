// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content should render the content 1`] = `
<DocumentFragment>
  <div
    class="contents"
  >
    <div
      class="content-heading"
    >
      <h1>
        Content Types
      </h1>
    </div>
    <div
      class="content-body"
    >
      <div
        class="content-list"
      />
      <div
        class="content-builder"
      />
    </div>
  </div>
</DocumentFragment>
`;
